app/views/admin/facilities/index.html.erb

```diff
@@ -28,13 +28,17 @@
         <td><%= facility.country %></td>
         <td><%= facility.pin %></td>
         <td><%= facility.facility_type %></td>
-        <td><%= link_to 'Edit', edit_admin_facility_path(facility) %></td>
-        <td><%= link_to 'Destroy', [:admin, facility], method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <% if policy(facility).edit? %>
+          <td><%= link_to 'Edit', edit_admin_facility_path(facility) %></td>
+          <td><%= link_to 'Destroy', [:admin, facility], method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
We should check if the user has destroy policy as a separate condition

Reply:
👍